<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditUser</class>
 <widget class="QWidget" name="AddEditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="lastnameEdit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>370</y>
     <width>811</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Entrer le nom de l'utilisateur ici</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="firstnameEdit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>560</y>
     <width>811</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Entrez le prénom de l'utilisateur ici</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthDateEdit">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>770</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lastnameLabel">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>300</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstnameLabe">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>490</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLabel" name="birtDateLabel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>690</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validationButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>920</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>920</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 4);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="topBorder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1921</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
background-color: rgb(255, 170, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>291</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>data/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1921</width>
     <height>1081</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 213, 213);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>920</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Supprimer l'utilisateur</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>lastnameEdit</zorder>
  <zorder>firstnameEdit</zorder>
  <zorder>birthDateEdit</zorder>
  <zorder>lastnameLabel</zorder>
  <zorder>firstnameLabe</zorder>
  <zorder>birtDateLabel</zorder>
  <zorder>validationButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>topBorder</zorder>
  <zorder>icon</zorder>
  <zorder>deleteButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
